src/views/product-integrations-landing/components/searchable-integrations-list/index.tsx

```diff
@@ -112,6 +112,7 @@ export default function SearchableIntegrationsList({
 		setTiersCheckedArray(tiersCheckedArray.map((v, i) => false))
 		setComponentCheckedArray(componentCheckedArray.map((v, i) => false))
 		setFlagsCheckedArray(flagsCheckedArray.map((v, i) => false))
+		setTypesCheckedArray(typesCheckedArray.map((v, i) => false))
```

Comment:
This needed to be added as the reset filters button was not doing anything when selecting a Type.

![CleanShot 2023-05-17 at 16 25 24@2x](https://github.com/hashicorp/dev-portal/assets/2105067/da8805f9-5d0f-4cfd-827c-17e82fee01f6)